public_api/src/machine_registry.ts

```diff
@@ -14,6 +18,7 @@ import { ipcInvoke, ipcSend } from "./ipc_wrapper.js";
 const UPDATE_UI_HANDLER_SUFFIX = "__h0";
 const RECIEVE_HANDLER_SUFFIX = "__h1";
 const ON_BUTTON_PRESSED_EVENT_SUFFIX = "__e0";
+const NETWORK_STAT_EVENT_SUFFIX = "__i0";
```

Comment:
Ah ok, this format should probably be documented? I kinda just made it up following the patterns of what you were doing and it's unclear what this is even for to me still xD

Reply:
yeah, you're right. I'll document it soon.

to make it clearer, for example, when the `updateUi` listener is created, the event ID will be the machine ID + `UPDATE_UI_HANDLER_SUFFIX` so for `example:example_generator`, the event ID will be `example:example_generator__h0` for "example generator handler 0" and for on button pressed it will be `example:example_generator__e0` for "example generator event 0"